docs/sql/general/constraints.rst

```diff
@@ -53,6 +53,104 @@ constraint or a table constraint.
 For further details about the meaning of the options see
 :ref:`indices_and_fulltext`.
 
+.. _check_constraint:
+
+``CHECK``
+---------
+
+The CHECK constraint specifies that the value/s of certain column/s must satisfy
+a boolean expression on insert and update.
+
+Syntax:
+
+::
+
+  [CONSTRAINT <name>] CHECK (boolean_expression)
+
+If `CONSTAINT <name>` is omitted, CrateDB generates a unique name automatically.
+This name is visible in :ref:`information_schema` and can be used to
```

Comment:
please correct the `ref` to point to the relevant table.. sorry I was too lazy in the suggestion to look it up.